### PR TITLE
Add optimizations to Blazar calendar

### PR DESCRIPTION
**Pull request by super-cooper**
Added some configuration parameters to the chart generator which should resolve some performance issues with the calendar. See https://github.com/apexcharts/apexcharts.js/issues/214 for some more context.

### CODE REVIEW

**Comment by super-cooper** [comment]
[Dashboard - The calendar is very slow on Chrome](https://trello.com/c/Xbn34FoB/2630-dashboard-the-calendar-is-very-slow-on-chrome)

File: blazar_dashboard/static/leases/js/calendar/lease_chart.js [hunk 1]
```diff
@@ -246,6 +250,13 @@
             }
           ]
         },
+        // Free performance
+        animations: {
+          enabled: false
+        },
+        dataLabels: {
```
Comment (Mark-Powers) [inline blazar_dashboard/static/leases/js/calendar/lease_chart.js:257]: Does this change the way the calendar looks at all?

**Review by Mark-Powers — APPROVED** [review]
Looks good! Does the calendar look the same/is as functional as before? I assume it is good, but unsure if we used data labels before, or if the animations were doing much.

File: blazar_dashboard/static/leases/js/calendar/lease_chart.js (hunk 1 above)
Comment (super-cooper) [reply blazar_dashboard/static/leases/js/calendar/lease_chart.js:257]: It doesn't look any different to me
image

Comment (Mark-Powers) [reply blazar_dashboard/static/leases/js/calendar/lease_chart.js:257]: @super-cooper Did you check how it looks with active allocations in the calendar? I wonder if they are a source of lag. You could just reserve all of them at once.

Comment (super-cooper) [reply blazar_dashboard/static/leases/js/calendar/lease_chart.js:257]: image
looks good to me :)

**Comment by super-cooper** [comment]
@Mark-Powers if you could, please look at the changes I made to requirements.txt. They seem obvious to me, but I am wondering if there may have been a reason for that file to be the way that it was.

**Comment by Mark-Powers** [comment]
I think requirements.txt is OK, it looks similar to what we had historically. The biggest concern would just be to build and test this on dev after deploying. I would change the branch to `chameleoncloud/xena` though, since that is what we will use going forward.

**Pull request merged**